package.json

```diff
@@ -46,9 +45,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/osano/cookieconsent.git"
+    "url": "https://github.com/sandstreamdev/cookieconsent"
+  },
+  "author": {
```

Comment:
Can we list multiple authors and point to the original repo as well?

Reply:
We could leave the old name and add the contributors link as it is right now.

About the repository link - it is link to our fork so if someone will open it there will be information about it

Reply:
Author is reverted to original value.